cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -0,0 +1,133 @@
+import * as addDisasterUtil from '../../../docs/import/add_disaster_util.js';
+import {populateColorFunctions, withColor} from '../../../docs/import/color_function_util.js';
+import {createTrs, setDisasterAndLayers} from '../../support/import_test_util.js';
+import {addFirebaseHooks, loadScriptsBeforeForUnitTests} from '../../support/script_loader.js';
+
+describe('Unit tests for add_disaster page', () => {
+  loadScriptsBeforeForUnitTests('ee', 'firebase', 'jquery');
+  addFirebaseHooks();
+
+  const property = 'color-function';
+  let colorFunctionEditor;
+  let writeToFirebaseStub;
+
+  before(() => {
+    cy.wrap(firebase.auth().signInWithCustomToken(firestoreCustomToken));
+
+    colorFunctionEditor =
+        $(document.createElement('div')).prop('id', 'color-fxn-editor').hide();
+    colorFunctionEditor.append(
+        makeTypeDiv('single'), makeTypeDiv('continuous'),
+        makeTypeDiv('discrete'));
+    $(document.body).append(colorFunctionEditor);
+
+    populateColorFunctions();
+
+    /**
```

Comment:
nit: just a style preference, but pull this out to the top level if it doesn't need to be nested?